opds.py

```diff
@@ -539,7 +539,7 @@ def page(cls, _db, title, url, lane, annotator=None,
         for args in cls.facet_links(annotator, facets):
             OPDSFeed.add_link_to_feed(feed=feed.feed, **args)
 
-        if len(works) > 0:
+        if len(works) > 0 and pagination.has_next_page:
```

Comment:
This change has the potential to break controller tests in circulation, the metadata wrangler, and may also impact the clients. It removes the "next" link if there aren't any more works in the query.